lib/AST/ASTImporter.cpp

```diff
@@ -3363,6 +3363,30 @@ getFriendCountAndPosition(FriendDecl *FD) {
   return std::make_tuple(FriendCount, *FriendPosition);
 }
 
+DeclContext* getDCOfUnderlyingDecl(FriendDecl *FrD) {
```

Comment:
should this be `static`?

Reply:
Yes: I found similar function that are static (`setTypedefNameForAnonDecl`).